test/unit/import_playbooks_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'test_plugin_helper'
+require_relative '../../app/services/foreman_ansible/playbooks_importer'
+require_relative '../fixtures/output_example_from_playbooks_importer'
+
+class ImportPlaybooksTest < ActiveSupport::TestCase
+  setup do
+    @actual_importer = ::ForemanAnsible::PlaybooksImporter.new(@proxy)
+    @playbooks_names = ['xprazak2.forklift_collection.collect_debug_draft.yml']
+    @importer = mock('Playbook Importer')
+    @proxy_api = mock('ProxyAPI')
+    @proxy = FactoryBot.create(:smart_proxy, :with_ansible)
+    @params_create = OutputExample.params_create
+    @params_update_no_inputs = OutputExample.params_update_no_inputs
+    @params_update = OutputExample.params_update
+    sample_playbooks = JSON.parse(File.read(ansible_fixture_file('sample_playbooks.json')))
+    @playbook1_output = sample_playbooks['1']
+    @playbook2_output = sample_playbooks['2']
```

Comment:
These are used in just few tests each, but initialized for every test (as setup is run for all of them).
It would be better to actually eighter:

1. initialize these just in the tests that are using them or,
2. use `let(:param1) { param_definition }` syntax, that initializes the parameters as local variables once called eg:
```
let(:sample_playbooks) { JSON.parse(File.read(ansible_fixture_file('sample_playbooks.json'))) }
let(:playbook1_output { sample_playbooks['1'] }

test '' do
  expects(playbook1_output).to be_kind_of(Hash)
end